entries/i/independentreserve.com.json

```
@@ -0,0 +1,43 @@
+{
+  "Independent Reserve": {
+    "domain": "independentreserve.com",
+    "tfa": [
+      "totp"
```

SMS ist not recommended, but it's available nonetheless.
```suggestion
      "totp",
      "sms"
```